pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "llama-stack-provider-trustyai-fms"
-version = "0.1.4"
+version = "0.1.5"
```

Comment:
If we're planning on syncing upstream provider version with downstream LLS releases, should we bump this to `0.2.0`?